pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -231,3 +232,28 @@ var UseMuxRangeFeed = settings.RegisterBoolSetting(
 	"if true, changefeed uses multiplexing rangefeed RPC",
 	false,
 )
+
+// EventConsumerWorkers specifies the maximum number of workers to use when
+// processing  events.
+var EventConsumerWorkers = settings.RegisterIntSetting(
+	settings.TenantWritable,
+	"changefeed.event_consumer_workers",
+	"the number of workers to use when processing events; 0 disables",
+	int64(util.ConstantWithMetamorphicTestRange("changefeed.consumer_max_workers", 0, 0, 32)),
+	settings.NonNegativeInt,
+).WithPublic()
+
+// EventConsumerWorkerQueueSize specifies the maximum number of events a worker buffer.
+var EventConsumerWorkerQueueSize = settings.RegisterIntSetting(
+	settings.TenantWritable,
+	"changefeed.event_consumer_worker_queue_size",
+	"if changefeed.event_consumer_workers is enabled, this setting sets the maxmimum number of events"+
+		"which a worker can buffer",
+	16,
```

Comment:
let's make that metamorphic... 0 -> 16?
Also.... 16 was rather arbitrary, no?

Reply:
It's arbitrary. I feel like setting a size of 0 or 1 is iffy. It might cause slowdowns. Making it unbounded is a bit scary. I'll make it metamorphic for 4-16.

Reply:
Turns out, using 0 for size is great at finding bugs...

Reply:
Sure. I'll do it. But, I will look forward to setting it to my scientifically chosen value of 16 in the next month or so 😄